.gitignore

```diff
@@ -1 +1,4 @@
 __pycache__/
+.idea/
+.vs/
+build/
```

Comment:
Pls ignore this, I've branched off **gitignore_rules**
I see these changes are already merged to originating repo

Reply:
Please ask advice @dkurt. He is owner repo. Because rewriting gitignore's file does not include in our rules.

Reply:
> I see these changes are already merged to originating repo
These changes are already merged into original repo, please checkout

Reply:
Ok, thank you, I saw. You need to update yours branch.